tests/test_parser.rs

```diff
@@ -0,0 +1,15 @@
+use prql::{ast_of_string, Result, Rule};
+
+#[test]
+fn parse_simple_string_to_ast() -> Result<()> {
+    assert_eq!(
+        serde_yaml::to_string(&ast_of_string("select 1", Rule::query)?)?,
```

Comment:
Very nice!

FYI if we use `insta`, we get the value added in "for free" (but no need to change)